doc/architecture/compute/operator-service-endpoints.md

```diff
@@ -0,0 +1,92 @@
+# Compute Service Endpoints
+
+## Status
+  
+  
+### GET /api/v1/operator/status
+   
+   
+Get all workflows and corresponding stats
+
+Parameters
+```
+        owner: String object containg user ETH address (optional)
+        agreementID: String object containing agreementID (optional)
+        workflowID: String object containing workflowID (optional)
+```
+
+Returns
+
+An Array of objects, each object describing a workflow. If the array is empty, then the search yeld no results
```

Comment:
so I pass in a unique workflow ID, but get multiple workflows back? Isn't every workflow supposed to have one unique ID?

Reply:
No, you will only get one workflow.
But if you search using only ownerID, you will get multiple workflows if user has multiple jobs